src/java.desktop/share/classes/sun/java2d/marlin/TransformingPathConsumer2D.java

```diff
@@ -898,22 +899,22 @@ public long getNativeConsumer() {
         final float[] clipRect;
 
         // clip rectangle (ymin, ymax, xmin, xmax) including padding:
-        final float[] clipRectPad = new float[4];
+        final double[] clipRectPad = new double[4];
```

Comment:
are all the changes to double needed for this ? Or just preparation for removing the float pipeline ?

Reply:
These change are just enough to make MarlinRenderingEngine (float) pass the new test.
Later we will propose to remove the float pipeline.